plan/policy.go

```diff
@@ -26,6 +26,8 @@ var Policies = map[string]Policy{
 	"sync":        &SyncPolicy{},
 	"upsert-only": &UpsertOnlyPolicy{},
 	"create-only": &CreateOnlyPolicy{},
+	"first-half":  &FirstHalfChangesPolicy{},
+	"last-half":   &LastHalfChangesPolicy{},
```

Comment:
This map is only used for making the _policies_ configurable via the command line flag `--policy`. I don't think we want to expose them directly to the user and therefore can remove here.